packages/react-sdk/src/components/CallLayout/PaginatedGridLayout.tsx

```diff
@@ -121,22 +165,22 @@ const chunk = <T extends unknown[]>(array: T, size = GROUP_SIZE) => {
 // problem is, participants change due to anything (someone muted video for example)
 // which would yield new merged stream anytime someone interacted with something
 // completely unrelated to published audio tracks
-const useMergeAudioStreams = (participants: StreamVideoParticipant[]) => {
-  return useMemo(() => {
-    const audioStreams = participants
-      .map((p) => p.audioStream)
-      .filter(Boolean) as MediaStream[];
-
-    const audioContext = new AudioContext();
-
-    const sourceNodes = audioStreams.map((stream) =>
-      audioContext.createMediaStreamSource(stream),
-    );
-
-    const dest = audioContext.createMediaStreamDestination();
-
-    sourceNodes.forEach((sourceNode) => sourceNode.connect(dest));
-
-    return dest.stream;
-  }, [participants]);
-};
+// const useMergeAudioStreams = (participants: StreamVideoParticipant[]) => {
+//   return useMemo(() => {
+//     const audioStreams = participants
+//       .map((p) => p.audioStream)
+//       .filter(Boolean) as MediaStream[];
+//
+//     const audioContext = new AudioContext();
+//
+//     const sourceNodes = audioStreams.map((stream) =>
+//       audioContext.createMediaStreamSource(stream),
+//     );
+//
+//     const dest = audioContext.createMediaStreamDestination();
+//
+//     sourceNodes.forEach((sourceNode) => sourceNode.connect(dest));
+//
+//     return dest.stream;
+//   }, [participants]);
+// };
```

Comment:
Let's get rid of this thing completely, it's untested and wouldn't work in a "performant" manner anyway.

Reply:
Fine with me :)

Reply:
done.